### ISSUE TITLE
Portal does not display IPs for non-bootstrapped nodes

### ISSUE THREAD

**Issue by gmiranda23** [chef-training/chef_classroom#3]
Right now the portal only displays instance IDs as a POC for how the portal works. We need to muck with the `aws_object` method to pull out IPs for machines that haven't ever run chef-client and stuff those somewhere useful. The approach will be some sort of method similar to this:

```
machine_image 'my_image' do
      ...
end

ruby_block "look up machine_image object" do
  block do
    aws_object = Chef::Resource::AwsImage.get_aws_object(
      'my_image',
      run_context: run_context,
      driver: run_context.chef_provisioning.current_driver,
      managed_entry_store: Chef::Provisioning.chef_managed_entry_store(run_context.cheffish.current_chef_server)
    )
  end
end
```

**Comment by gmiranda23** [comment]
I'm closer to getting this done.  In the current `deploy_multi_nodes.rb` recipe, I can add the following code to get all the values needed.

```
1.upto(node2_count) do |i|
  log "#{name}-node2#{i} info" do
    message lazy {
    aws_object = Chef::Resource::AwsInstance.get_aws_object(
      "#{name}-node2#{i}",
      run_context: run_context,
      driver: run_context.chef_provisioning.current_driver,
      managed_entry_store: Chef::Provisioning.chef_managed_entry_store(run_context.cheffish.current_chef_server)
    )
    "the ip of #{name}-node2#{i} is #{aws_object.public_ip_address}"
    }
  end
end
1.upto(node3_count) do |i|
  log "#{name}-node3#{i} info" do
    message lazy {
    aws_object = Chef::Resource::AwsInstance.get_aws_object(
      "#{name}-node3#{i}",
      run_context: run_context,
      driver: run_context.chef_provisioning.current_driver,
      managed_entry_store: Chef::Provisioning.chef_managed_entry_store(run_context.cheffish.current_chef_server)
    )
    "the ip of #{name}-node3#{i} is #{aws_object.public_ip_address}"
    }
  end
end
```

I'm not entirely sure yet how best to get these to the portal node.  Perhaps template variables can be passed in as lazy that do this lookup?  Or should these get stuffed into the run_context somewhere?  This requires some digging, but the info is definitely accessible.

**Comment by gmiranda23** [comment]
I'm leaving this open even though it's resolved by PR #17.  There has to be a better way.

**Comment by gmiranda23** [comment]
Addressed by #17.  That's actually not a bad approach.
